src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/TestData.kt

```diff
@@ -1,39 +1,44 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
 
-import org.mockito.kotlin.mock
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.CommunityOffenderManager
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.CrdLicence
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Licence
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.StandardCondition
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.VariationLicence
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
 import java.time.LocalDate
 import java.time.LocalDateTime
 
 object TestData {
 
-  val someEntityStandardConditions = listOf(
-    StandardCondition(
-      id = 1,
-      conditionCode = "goodBehaviour",
-      conditionSequence = 1,
-      conditionText = "Be of good behaviour",
-      licence = mock(),
-    ),
-    StandardCondition(
-      id = 2,
-      conditionCode = "notBreakLaw",
-      conditionSequence = 2,
-      conditionText = "Do not break any law",
-      licence = mock(),
-    ),
-    StandardCondition(
-      id = 3,
-      conditionCode = "attendMeetings",
-      conditionSequence = 3,
-      conditionText = "Attend meetings",
-      licence = mock(),
-    ),
-  )
+  fun someEntityStandardConditions(licence: Licence) =
```

Comment:
setting back reference to owning licence correctly (did this across a load of tests)